Automattic/wp-calypso
# Add Tracks event to ccpa opt-out click

#### Proposed Changes

* Add `a8c_ccpa_optout` to exceptions that do not use `calypso_` prefix
* Implement `a8c_ccpa_optout` on CCPA opt-out

#### Testing Instructions

* Checkout this PR and run `yarn start`
* Enter `http://calypso.localhost:3000/me/privacy?flags=cookie-banner,ad-tracking`
* Ensure you've set cookies `region=Colorado`, `country_code=US`
* Refresh the page, you should see CCPA setting
* Click on toggle to opt-out from advertising
* You should get Tracks event `a8c_ccpa_optout`

#### Pre-merge Checklist





Related to #
